src/components/States/States/Empty.tsx

```diff
@@ -0,0 +1,260 @@
+import React from "react";
+import { IState, State } from "./State";
+import { TeamsTheme } from "../../../lib/withTheme";
+
+const DEFAULT_STRINGS = {
+  title: `You don’t have any tasks yet`,
+  desc: `Once you are assigned a task, you can find it here. Until then, enjoy your time off.`,
+};
+
+const Empty = ({ theme, values }: { theme: TeamsTheme; values?: IState }) => {
+  let config: IState = DEFAULT_STRINGS;
+  if (values) {
+    config = { ...config, ...values };
+  }
+  return <State image={illustrations[theme]} {...config} />;
+};
+
+const illustrations = {
+  [TeamsTheme.Default]: (
+    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 252 182.2">
```

Comment:
If we're packaging possible SVGs with these, we should make sure none of the SVGs have people in them

Reply:
Did we find a replacement for the people SVG?